src/utils/csvParser/parser.test.ts

```diff
@@ -0,0 +1,1544 @@
+/* eslint-disable @typescript-eslint/no-loss-of-precision */
+import {expect, jest} from '@jest/globals'
+import {CSVParser} from './parser'
+import {EOFError} from './utils'
+
+interface Metric {
```

Comment:
rather than re-creating this here and in `parser.ts`, just export it from `parser.ts` and import it here

Reply:
good call